### ISSUE TITLE
libtiff: update to 4.5

### ISSUE THREAD

**Issue by shawnlaffan** [StrawberryPerl/build-extlibs#22]
libtiff version 4.5 has been released.  https://libtiff.gitlab.io/libtiff/releases/v4.5.0.html

build-extlibs should only need changes to the downloads and build lists to get it included.  

libtiff 4.5 builds without issue using Alien::libtiff under SP 5.32, although the alien does not run the libtiff test suite.  

I'll submit a PR when I get a chance to test it && no-one has beaten me to it.

**Comment by shawnlaffan** [comment]
Version 4.5 builds but fails four tests.  These all relate to the tiff2ps utility which is not normally distributed with Strawberry Perl so might not actually matter (unless of course it is exposing underlying issues).  

```
FAIL: tiff2ps-PS1.sh
FAIL: tiff2ps-PS2.sh
FAIL: tiff2ps-PS3.sh
FAIL: tiff2ps-EPS1.sh
```

**Comment by shawnlaffan** [comment]
The test failures were a line ending issue that the MSYS2 project have already developed a patch for.  This is included in PR #24
